app/controllers/apikey_controller.rb

```diff
@@ -0,0 +1,15 @@
+class ApikeyController < ApplicationController 
```

Comment:
Consider if you'd like to perform a `before_action` authentication check on this controller. See the `before_action` in the **original** `TodosController` (not the API one) that used devise's `authenticate_user!` method.